add_up.rb

```diff
@@ -0,0 +1,16 @@
+#(3) Write a program with a function add_up(i) .  
+#It is to be passed a positive integer, and it will add all the numbers from 1 to that integer and return the sum.  
+#Call the function three times within the program, and each time print out the return value.  
+#Call the program add_up.rb.
+
+def add_up(i)
+    sum = number
```

Comment:
I'm guessing that this one went through a bunch of revisions! `number` is not defined anywhere in the function. I think based on the rest of your code you should initialize `sum` to start at zero (a blank slate!)